src/Apps/Artwork/Components/Pageviews.tsx

```diff
@@ -0,0 +1,102 @@
+import { color, Sans } from "@artsy/palette"
+import { Pageviews_artwork } from "__generated__/Pageviews_artwork.graphql"
+import { PageviewsQuery } from "__generated__/PageviewsQuery.graphql"
+import { ContextConsumer } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { PageviewsIcon } from "Styleguide/Elements/icons/Pageviews"
+
+interface Props {
+  artwork: Pageviews_artwork
+}
+
+export class Pageviews extends Component<Props> {
+  render() {
+    const { artwork } = this.props
+    const { pageviews } = artwork
+
+    if (pageviews) {
+      return (
+        <>
+          <PageviewsIcon width="21" height="21" />
+          <div
+            style={{
+              display: "inline-block",
+              verticalAlign: "sub",
+              marginLeft: "15px",
+            }}
+          >
+            <Sans weight="medium" size="2">
+              This work is getting noticed!
+            </Sans>
+            <Sans color={color("black60")} size="2" weight="regular">
+              It has been viewed{" "}
+              <span style={{ fontWeight: "bold" }}>
+                {pageviews.toLocaleString()}
+              </span>{" "}
```

Comment:
no span, use our fonts...

Reply:
this just bolds a word within a `Sans` tho...I rewrote the other parts to use `Flex` and `Box` (which btw was way easier/nicer than my initial attempt), but I'm not sure how to rewrite this?

Reply:
`<Sans weight='medium'>{pageviews.toLocaleString()}</Sans>` (or bold if that's available as a weight)

Reply:
Or even just `<strong>something</strong>`